cli/pom.xml

```diff
@@ -156,8 +156,8 @@
         </executions>
       </plugin>
       <plugin>
```

Comment:
Do we still need to invoke it explicitly? Parent POm should be doing it

Reply:
> Do we still need to invoke it explicitly? Parent POm should be doing it

Probably not. But if so, this was a pre-existing issue and not something caused by the switch to SpotBugs.

Reply:
I went ahead and removed this to clean up this code.